bobber/__version__.py

```diff
@@ -1,2 +1 @@
-# SPDX-License-Identifier: MIT
```

Comment:
Is this intentional?

Reply:
It was intentional originally, yes, as the script didn't play well with it, but I found a better way to include it that I will update it with.